tests/wpt/mozilla/meta/mozilla/service-workers/service-worker-registration.https.html.ini

```diff
@@ -1,34 +1,12 @@
 [service-worker-registration.https.html]
-  [Test: Asserts Installing Service Worker and its Registration]
```

Comment:
The updates here are unexpected, did you notice they were consistently passing locally?

Reply:
Indeed, it is failing. I was misled by CI. I will revert this